plugins/kopilot.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kopilot
+spec:
+  version: v0.0.2
+  homepage: https://github.com/knight42/kopilot
+  shortDescription: Your AI Kubernetes expert who can help you diagnose and audit resources.
```

Comment:
Majority of this will get truncated in the cli (max 50 chars). What do you think of:

> Diagnose/audit resources with AI

I am trying to make sure the value add is clear to users. 

Maybe the description is a good place to explain where users' data is uploaded to.

Reply:
SGTM, please take another look.

Reply:
gently ping @ahmetb